src/pages/Lend/LoanChannelCategoryClimateExperiment.vue

```diff
@@ -131,36 +107,51 @@ export default {
 		},
 	},
 	apollo: {
-		preFetch: true,
-		query: ecoCategoryPageQuery
+		preFetch(config, client, args) {
+			const { route } = args;
+			const { params } = route;
+
+			// Filter routes on route.param.category to get current path
+			const targetedLoanChannelURL = params.category;
+
+			// Isolate targeted loan channel id
+			const targetedLoanChannelID = loanChannelQueryMapMixin.data().loanChannelQueryMap
+				.find(channel => channel.url === targetedLoanChannelURL).id;
```

Comment:
Does there need to be optional chaining on `.id`?

Reply:
I can add these to be extra safe, but going forward these channels should _really_ be in the query-map